internal/integration/blocks_test.go

```diff
@@ -231,26 +231,23 @@ func AssertBlockKubernetesDeploymentCreateAndRunning(ctx context.Context, manage
 }
 
 // AssertClusterCreateAndReady is a reusable group of tests that can be used to verify that a cluster is created and ready.
-func AssertClusterCreateAndReady(ctx context.Context, rootClient *client.Client, name string, options ClusterOptions) []subTest { //nolint:nolintlint,revive
-	clusterName := "integration-" + name
-	options.Name = clusterName
-
+func AssertClusterCreateAndReady(ctx context.Context, rootClient *client.Client, options ClusterOptions) []subTest { //nolint:nolintlint,revive
```

Comment:
For the reviewer: This refactor was my suggestion, so that we have a single way of passing cluster name. Not related otherwise to the PR.